Makefile

```diff
@@ -16,20 +16,24 @@ all:
 # spec .JSON files to the project
 project:
 	swift package generate-xcodeproj
-	@$(call check_for_gem, xcodeproj)
+	@$(call check_for_gem,xcodeproj)
 	ruby add_json_files.rb
 
 test:
 	swift test -v $(FILTERARG)
 
 test-pretty:
-	@$(call check_for_gem, xcpretty)
-	swift test $(FILTERARG) 2>&1 | xcpretty
+	@$(call check_for_gem,xcpretty)
+	set -o pipefail && swift test $(FILTERARG) 2>&1 | xcpretty
```

Comment:
this is so that when we run on Travis, `test-pretty` exits with a failure code if `swift test` fails.